docGen/nvd.go

```diff
@@ -481,6 +480,22 @@ func parseVulnerabilityJSONFile(fileName string) (VulnerabilityPost, error) {
 	}, nil
 }
 
+func detectVersion(includeVersion, excludeVersion, itemVersion string) string {
```

Comment:
I used NVD logic (`(including)` and `(excluding)` suffixes).
Tell me if you see better way.